lib/events/filesessions/fileasync_chaos_test.go

```diff
@@ -123,6 +123,7 @@ func TestChaosUpload(t *testing.T) {
 		ScanPeriod: scanPeriod,
 		Streamer:   faultyStreamer,
 		Clock:      clock,
+		AuditLog:   &events.AuditLog{},
```

Comment:
If it's not meant to be functional - you can use `event.NewDiscardAuditLog` instead.